src/mscorlib/shared/System/String.Searching.cs

```diff
@@ -19,6 +19,10 @@ public bool Contains(string value, StringComparison comparisonType)
             return (IndexOf(value, comparisonType) >= 0);
         }
 
+        public bool Contains(char value)
+        {
+            return IndexOf(value) != -1;
+        }
         // Returns the index of the first occurrence of a specified character in the current instance.
         // The search starts at startIndex and runs thorough the next count characters.
```

Comment:
this is missing 2 of the 3 new API, from https://github.com/dotnet/corefx/issues/25094:

        public bool Contains(char value) { throw null; }
        public bool Contains(char value, StringComparison comparisonType) { throw null; }
        public int IndexOf(char value, StringComparison comparisonType) { throw null; } // to implement above